src/action-creators/toggleToolbarAndBreadCrumbs.ts

```diff
@@ -0,0 +1,19 @@
+import { store } from '../store'
+import { ActionCreator } from '../types'
+
+/**
+ * Dispatches an alert action.
+ *
+ * @param value A boolean to represent the visibility state of toolbar & breadcrumbs.
+ */
+const toggleToolbarAndBreadCrumbs = (value: boolean): ActionCreator => dispatch => {
```

Comment:
The second argument after `dispatch` is `getState`. Use that to avoid accessing the `store` directly.

Reply:
I should've noticed. Will update